systemd/systemd.go

```diff
@@ -615,44 +615,50 @@ func (s *systemd) getPropertyStringValue(unit, key string) (string, error) {
 	return strings.TrimSpace(splitVal[1]), nil
 }
 
-func (s *systemd) CurrentTasksCount(unit string) (int, error) {
-	tasksStr, err := s.getPropertyStringValue(unit, "TasksCurrent")
+func (s *systemd) getPropertyUintValue(unit, key string) (uint64, bool, error) {
```

Comment:
(bikeshed) Maybe just return `(uint64, err)` here and have something like PropertyUnavailableError or just ErrNotExists or something?